### ISSUE TITLE
t_global cannot be concatenate with styles

### ISSUE THREAD

**Issue by yasohasakii** [JiauZhang/GigaGAN#7]
When use_text_cond is True , the text encoder works but text embeds cannot be concatenate with image features in model.py `styles = [torch.cat([style_, t_global], dim=1) for style_ in styles]` , these tensors are not in same size, how should I modify this problem, thanks.
